src/main/kotlin/org/jitsi/rtp/rtcp/rtcpfb/transport_layer_fb/tcc/RtcpFbTccPacket.kt

```diff
@@ -41,9 +41,12 @@ import org.jitsi.rtp.util.getShortAsInt
 // Valid values are 0 (packet wasn't received), 1 or 2.
 typealias DeltaSize = Int
 
-class ReceivedPacket(val seqNum: Int, val deltaTicks: Short) {
+class ReceivedPacket(val seqNum: Int, val deltaTicks: Short, val received: Boolean = true) {
     operator fun component1(): Int = seqNum
     operator fun component2(): Short = deltaTicks
+    operator fun component3(): Boolean = received
+
+    constructor(seqNum: Int) : this(seqNum, 0, false)
```

Comment:
this looks a bit odd at first glance, i think a comment here that describes this is used for unreceived packets (which have no delta, therefore we hard-code 0 and the fact that it's unreceived, as this would only apply to an unreceived packet--something like that?)